### DISCUSSION TITLE
How can we run pytest files in parallel?

### DISCUSSION THREAD

**Discussion by thakkardharmik** [pytest-dev/pytest#8727] [Q&A]
How can we run pytest files in parallel?
I want to run multiple pytest files in parallel. pytest-xdist allows running tests in parallel but I believe it will run test methods in parallel. Is it possible to run each pytest file in parallel?

**Comment by nicoddemus** [comment]
Hi @thakkardharmik,

Did you try `pytest-xdist` with `--dist loadfile`?
